Repositories/ExperiencesRepository.cs

```diff
@@ -0,0 +1,66 @@
+using rexfinder_api.Migrations;
+using rexfinder_api.Models;
+using rexfinder_api.Repositories;
+
+public class ExperiencesRepository : IExperiencesRepository 
+{
+    private readonly MyPlacesDbContext _context;
+
+    public ExperiencesRepository(MyPlacesDbContext context)
+    {
+        _context = context;
+    }
+
+    public Experiences CreateExperience(Experiences newExperience)
+    {
+        _context.Experiences.Add(newExperience);
```

Comment:
We had this in MyPlaces. Do we want something like it here as well?
newExperience.CreatedOn = DateTime.Now.ToString();